dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -107,7 +109,6 @@ public final PreparedStatement createStatementWithBoundFixtureSymbols(
         String command = Options.isBindSymbols() ? parseCommandText(commandText) : commandText;
         PreparedStatement cs = getConnection().prepareStatement(
                 command);
-
```

Comment:
I think I must have added some debugging output and later removed it, along with this blank line.

I'll put it back.

(BTW, do we have any preferred style for when we want blank lines?)